agent/utils/utils_test.go

```diff
@@ -137,6 +150,19 @@ func TestGetRequestFailureStatusCode(t *testing.T) {
 	}{
 		{
 			name: "TestGetRequestFailureStatusCodeSuccess",
+			err: &awshttp.ResponseError{
+				ResponseError: &smithyhttp.ResponseError{
+					Response: &smithyhttp.Response{
+						Response: &http.Response{
+							StatusCode: http.StatusBadRequest,
+						},
+					},
+				},
+			},
+			res: 400,
+		},
+		{
+			name: "TestGetRequestFailureStatusCodeSuccess SDKv2",
```

Comment:
Test case names on L152 and L165 should be swapped (or alternatively, L154-L162 and L166-L167 should be swapped).